demo/index.html

```diff
@@ -334,10 +332,16 @@ <h1 class="text-3xl font-bold">
         },
       ];
 
+      const qs = window.location.search || '';
+      const darkMode = qs.indexOf('darkMode=true') > -1;
```

Comment:
Wouldn't be better if we add a button to toggle the dark mode option instead of manually change it in the url?

Reply:
Sure, I just didn't want to keep adding things that didn't match the basecamp example :)

I could add a separate demo for these things

Reply:
Good call